src/sfizz/Synth.cpp

```diff
@@ -345,6 +346,9 @@ bool sfz::Synth::loadSfzFile(const fs::path& file)
             if (!resources.filePool.preloadFile(region->sample, maxOffset))
                 removeCurrentRegion();
         }
+        else if (region->oscillator && !region->isGenerator()) {
```

Comment:
Should we add a "type" enum to Region?

Reply:
Yes, the reason of why I didn't is that I don't know what to call it

Reply:
Right... `source` with `sample`, `generator`, `wavetable` maybe is OK? or `dataSource`?

Reply:
Maybe designate `oscillator=on` and `sample=*something` as both oscillators.
And rename `region->oscillator` as something else.
(eg. `runAsOscillator`)

It reminds me: in the Cakewalk book, it's said that if a sample file is short enough, it's taken automatically as an oscillator.

Reply:
So what would be short enough? :D

Reply:
I thought substituting this conditional with something as `region->isOscillator()`, which is true under either these 2 conditions.